tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/WPFormsTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @package   Google\Site_Kit
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Tests\Core\Conversion_Tracking\Conversion_Event_Providers;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Event_Providers\WPForms;
+use Google\Site_Kit\Tests\TestCase;
+
+class WPFormsTest extends TestCase {
+
+	/**
+	 * WPForms instance.
+	 *
+	 * @var WPForms
+	 */
+	private $wpforms;
+
+	public function set_up() {
+		parent::set_up();
+		$this->wpforms = new WPForms( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+	}
+
+	public function test_is_active() {
+		$this->assertFalse( $this->wpforms->is_active() );
+		define( 'WPFORMS_VERSION', 1 );
```

Comment:
Let's make sure the definition here doesn't affect other tests by running this in it's own process (`runInSeparateProcess`) or clean it up with [runkit](https://github.com/google/site-kit-wp/blob/15be138ba08f1b8692e422126588d6a822bc3cee/tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php#L251-L255).

Reply:
Updated.